components/brave_vpn/browser/connection/ikev2/win/utils.cc

```diff
@@ -369,7 +369,7 @@ RasOperationResult RemoveEntry(const std::wstring& entry_name) {
     return GetRasErrorResult(error_get_phone_book_path,
                              "GetPhonebookPath() from RemoveEntry()");
   }
-
+  DisconnectEntry(entry_name);
```

Comment:
don't need to check the result of this?

Reply:
there is not much to do as this is called during uninstallation process, added logs